CRM/Contact/BAO/ProximityQuery.php

```diff
@@ -378,7 +389,6 @@ public static function fixInputParams(&$input) {
             }
           }
         }
-        return NULL;
```

Comment:
The return type was not being used anywhere, and the docblock was previously being contradictory. void ultimately felt more appropriate.